### PR TITLE
1 jr create event web

### PR DESCRIPTION
**Pull request by Arcanedarkness**

### CODE REVIEW

File: Frontend/Frontend/Web/Controllers/EventController.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        public ActionResult Create(CreateEventViewModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View();
+            }
+
+            DateTime dt = model.Date + model.Time;
+
+            var evnt = service.CreateEvent(model.Title, model.Description, model.NumOfParticipants, model.PriceFrom, model.PriceTo, model.Location, dt, model.IsPublic);
```
Comment (Jensen96) [inline Frontend/Frontend/Web/Controllers/EventController.cs:36]: MORE LINES YOU FAT FUCK

Comment (Jensen96) [reply Frontend/Frontend/Web/Controllers/EventController.cs:36]: HILSEN NIKLAS

Comment (Jensen96) [reply Frontend/Frontend/Web/Controllers/EventController.cs:36]: GIT BLAME NIKLAS

**Review by Jensen96 — APPROVED** [review]

File: Frontend/Frontend/Web/Views/Event/Create.cshtml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            @Html.LabelFor(model => model.PriceTo, htmlAttributes: new { @class = "control-label col-md-2" })
+            <div class="col-md-10">
+                @Html.EditorFor(model => model.PriceTo, new { htmlAttributes = new { @class = "form-control" } })
+                @Html.ValidationMessageFor(model => model.PriceTo, "", new { @class = "text-danger" })
+            </div>
+        </div>
+
+        <div class="form-group">
+            <div class="col-md-offset-2 col-md-10">
+                <input type="submit" value="Create" class="btn btn-default" />
```
Comment (refnedergaard) [inline Frontend/Frontend/Web/Views/Event/Create.cshtml]: Change value in submit button to "Opret"

**Review by refnedergaard — APPROVED** [review]

**Pull request merged**